### ISSUE TITLE
Lovelace resource issues

### ISSUE THREAD

**Issue by johannesdeboer** [V2G-liberty/addon-v2g-liberty#40]
Loading lovelace: in v2g_liberty_package.yaml disables Dashboard Resources in Home Assistant.

By loading lovelace: here, other HACS (frontend) integrations stop working, since you cannot load resources via UI & Yaml simultaneously in HA.

**Comment by ArdJonker** [comment]
Hmm, I indeed did not test this. How have you fixed it now? Removed the resources from the package and added them via HACS?
What would be your ideal solution?

**Comment by johannesdeboer** [comment]
I tried removing the resources, but it didn't help. It seems as long as 

```
lovelace:
  mode: yaml
```

is mentioned somewhere in the configuration, the resources from the Dashboard > Resources pane are skipped completely.

I found a [threat in the HA community that helped](https://community.home-assistant.io/t/you-can-also-add-yaml-dashboards-when-your-main-dashboard-is-ui-configured/703944/11).

I changed
